packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -469,6 +469,11 @@ export default class TabAdvanced extends Component {
                           </span>
                         )}
                       </XoSelect>{' '}
+                      {migrationNetwork !== undefined && (
```

Comment:
We want to show the remove button even if the network is unknown.
```suggestion
                      {pool.otherConfig['xo:migrationNetwork'] !== undefined && (
```